.travis.yml

```diff
@@ -13,24 +13,31 @@ env:
   - TOXENV=django22
 
 matrix:
-  exclude:
-    - python: 2.7
-      env: TOXENV=django20
-    - python: 2.7
-      env: TOXENV=django21
-    - python: 2.7
-      env: TOXENV=django22
   include:
     - python: 3.5
       env: TOXENV=quality
     - python: 3.5
       env: TOXENV=docs
+    - python: 3.8
+      env: TOXENV=quality
+    - python: 3.8
+      env: TOXENV=docs
+    - python: 3.8
+      env: TOXENV=django22
```

Comment:
You shouldn't need to add this one explicitly, should already be in the test matrix due to the top-level `python:` and `env:` lists.